src/figuresClasses.ts

```diff
@@ -1,11 +1,99 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape = 'triangle';
+
+  color: string;
+
+  a: number;
+
+  b: number;
+
+  c: number;
+
+  constructor(color: string, a: number, b: number, c: number) {
+    this.color = color;
+
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('Side lengths must be more that zero');
+    }
+
+    const sortedSides = [a, b, c].sort((x, y) => x - y);
+
+    if (sortedSides[2] >= sortedSides[0] + sortedSides[1]) {
+      throw new Error(`Sides ${a}, ${b}, and ${c} can't form a triangle`);
```

Comment:
[CODE STYLE] - The error message should include a more detailed description. For example, 'The sum of any two sides of a triangle must be greater than the third side.'